Framework/Algorithms/src/ReflectometryWorkflowBase2.cpp

```diff
@@ -15,6 +17,23 @@ using namespace Mantid::Geometry;
 namespace Mantid {
 namespace Algorithms {
 
+/** Initialize properties related to the type of reduction
+*/
+void ReflectometryWorkflowBase2::initReductionProperties() {
+  // Summation type
+  std::vector<std::string> summationTypes = {"SumInLambda", "SumInQ"};
+  declareProperty("SummationType", "SumInLambda",
+                  boost::make_shared<StringListValidator>(summationTypes),
+                  "The type of summation to perform.", Direction::Input);
+
+  // Reduction type
+  std::vector<std::string> reductionTypes = {"None", "DivergentBeam",
```

Comment:
As mentioned above. It might be worth having those values available as `static const`. this way you don't have to write out strings in several places.